### DISCUSSION TITLE
三十六 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#449] [General]
# 三十六 - Miao Yu | 于淼

https://yufree.cn/cn/2023/12/31/36/

**Comment by yihui** [comment]
关于让你反胃的“情绪价值”一词，本丛林猴也是深度疑惑。上次你说[评分](https://yufree.cn/cn/2023/12/17/score/)时我就想回一句，真是搞不懂为啥过去一些模糊、定性的概念现在仿佛都有了个定量版本，比如颜“值”。

情绪有没有价值？当然有，我也承认情绪很重要。问题只是这些缥缈的概念稀里糊涂地被量化后，结果就造成了一种幻象，那就是它可以更简单地被优化，毕竟可以量化的东西我们是可以“清楚”看见优化的结果的。姑且不谈这种幻象有多没谱，它造成的次生结果是它可以和另一种可量化的指标绑在一起，就是钱。然后我也是活久见：[情绪价值都可以“投机”了](https://www.thepaper.cn/newsDetail_forward_25569341)。

**Reply by yufree** [reply]
这文章真大开眼界，虽说这种事一个愿打一个愿挨确实也是个人自由，但确实说明教育体系没有对这些话题进行良好引导，给了市场投机机会。备不住生成式人工智能也能做这事，国家可用开源框架免费提供一个面向所有人的心理咨询接口来打补丁，稍微拿点钱出来租个集群托管一下，这事要是个人做一定会被阴谋论黑，但就怕这伙人给人洗脑只有付费的或贵的才是好的，那外部能做的就真不多了。

**Reply by zocozocozoco** [reply]
搜了搜「情绪价值」最早是从经营营销学出来的，现在变到大众用词是很奇怪。国内大厂现在的确会经常提「情绪价值」这个词，偏贬义。一般称某某某员工主要价值是给领导提供情绪价值，意思是跪舔老板，或者说每天主要精力就是迎合上级，没有业务价值。

最近在看技术哲学有个研究点是现代人有「量化自我」的倾向，因为参照变多且复杂，人没办法衡量自己所处的位置和在社会的价值，可能会通过一种相对普遍认可的评价体系来衡量一些事情，或许也是「评分」和「情绪价值」出现的原因。

**Comment by zanteken** [comment]
新年好啊于淼，拜读你的文字一年多了，一直很有收获。感觉你运动提的不多，平时的运动是游泳吗？

**Reply by yufree** [reply]
第八套广播体操，或者步行一小时上班，基本木有以运动为目的的运动
